flareio/api_client.py

```diff
@@ -47,18 +47,22 @@ def _create_session() -> requests.Session:
             ),
         )
 
+        retry = Retry(
+            total=5,
+            backoff_factor=2,
+            status_forcelist=[429, 502, 503, 504],
+            allowed_methods={"GET", "POST"},
+        )
+
+        try:
```

Comment:
Can you check for python version instead?

Reply:
It's not a Python version issue tho.

Reply:
Yeah sure your suggestion is fine.